### ISSUE TITLE
Add "Cumulative Equivalent KLIMA " indicator so users can easily see their "KLIMA power"

### ISSUE THREAD

**Issue by 0xAurelius** [KlimaDAO/klimadao#379]
There has been a request from a community member to add an indicator on the app of how much KLIMA-equivalents are present total in the user's wallet

Since the unit of account for wsKLIMA isn't the same as for sKLIMA and KLIMA, it can be confusing and difficult for the user to assess how many "units" of KLIMA are present in their wallet
![image](https://user-images.githubusercontent.com/91024694/167305056-79cb7ae6-464a-4172-be25-1b9cd3a6acaa.png)

Basically add a "total" line to this type of display that converts wsKLIMA into its KLIMA equivalent if it were to be unwrapped

**Comment by Atmosfearful** [comment]
@0xAurelius @biwano These cards don't look great as they fill up with more and more items.

Also, since the /stake view can only interact with sklima <> klima, I don't think it even makes sense to show wsKLIMA here.
If they want to see their wsklima balance they can navigate to the /wrap view where we show the unwrapped equivalent balance already. 

This hints at a larger need for a general "dashboard" or "desk" view as I've teased in some mockups that I shared internally.

Closing this in favor of https://github.com/KlimaDAO/klimadao/issues/497
